src/cluster_legacy.c

```diff
@@ -7369,6 +7466,16 @@ const char **clusterCommandExtendedHelp(void) {
         "LINKS",
         "    Return information about all network links between this node and its peers.",
         "    Output format is an array where each array element is a map containing attributes of a link",
+        "IMPORT(-PREPARE) SLOTSRANGE <start slot> <end slot> [<start slot> <end slot> ...]",
+        "    Import the specified slot ranges from their owners.",
+        "IMPORT-COMMIT LINK <link-name>",
+        "    Commit a previous import started with IMPORT-PREPARE.",
+        "IMPORT-CANCEL ALL",
+        "    Cancel all ongoing imports.",
+        "MIGRATIONS",
+        "    Get information about ongoing and recently finished slot imports and exports.",
+        "SYNCSLOTS <subcommand>",
```

Comment:
This is an internal command, we do not need to print it as part of a message to expose to the user.